client/functions.lua

```diff
@@ -244,7 +244,7 @@ function QBCore.Functions.IsWearingGloves()
     local ped = PlayerPedId()
     local armIndex = GetPedDrawableVariation(ped, 3)
     local model = GetEntityModel(ped)
-    if model == `mp_m_freemode_01` then
+    if model == 'mp_m_freemode_01' then
```

Comment:
Change this back, this will make an error because it will try to compare a number to a string, backticks are used to make hashkeys, which are numbers

Reply:
Sorry about that